### DISCUSSION TITLE
Feature request to show limited number of posts

### DISCUSSION THREAD

**Discussion by gpoulter** [jordwest/news-feed-eradicator#427] [Ideas]
An interesting option, since the bulk of the wasted time is from scrolling the feed, is to show the first 1/5/10/20/50 feed items and eliminate the remainder.

This could be an alternative to pausing for X minutes, and would enable one to forget eventually that there ever was an infinite scroll.

It is likely be more difficult than eliminating the entire feed, due to deleting individual blocks of content or intercepting the scroll action that would trigger loading more.

Update: I've found the "Scroll Stop" extension which redirects one after scrolling a certain number of pixels, but still think finding the feed ending after a number of posts would provide a better experience.

**Comment by jordwest** [comment]
I don't think will ever be implemented sorry, as detecting individual feed elements and filtering them is a huge amount of ongoing maintenance especially for multiple websites
